packages/desktop-client/src/components/autocomplete/NewPayeeAutocomplete.js

```diff
@@ -154,7 +150,6 @@ export default function PayeeAutocomplete({
                 }
               }
               onClick={() => {
-                setInputValue('');
```

Comment:
This is a small downside to the current implementation. Theoretically we could duplicate all the `inputValue` manipulation logic in this file too, but.. that would be a lot of complex duplications I'd prefer not to do.

So I would recommend removing this line in favor of the greater good.